.travis.yml

```diff
@@ -1,7 +1,10 @@
+dist: xenial
```

Comment:
In general, we're supposed to switch to xenial linux, which has a [simpler set of steps](https://docs.travis-ci.com/user/database-setup/#mysql-57) for enabling MySQL support - Travis will auto-upgrade us to xenial soon ([link](https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment)) unless we pin `dist: trusty`. However xenial comes with JDK 11 ([link](https://docs.travis-ci.com/user/reference/xenial/#jvm-clojure-groovy-java-scala-support)), which PySpark doesn't seem to like (Spark tests are failing...). Will look into it tomorrow.